src/Pages/Signup-Page/signup.css

```diff
@@ -0,0 +1,119 @@
+.signup-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 4rem minmax(calc(100vh - 9rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+
+.signup__main {
+  padding-bottom: 4rem;
+  grid-area: main;
+  background-color: var(--primary-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* signup form Stats Here */
+
+.signup__title {
+  border-bottom: 2px solid var(--primary-color);
+  padding: 0.5rem;
+  text-align: center;
+  margin-bottom: 1rem;
+  color: var(--primary-color);
+  font-size: var(--font-size-X-large);
+}
+.signup-form {
+  margin: 0.5rem;
```

Comment:
suggestion: you can create your own utility class having margin, padding, width, flex properties etc and use that.

This will make your CSS code less and It will be easy for you too.

Reply:
Okay,I will create and use that classes.